src/fixup-svg-string.js

```diff
@@ -18,6 +18,13 @@ module.exports = function (svgString) {
             svgAttrs[0].replace(/&ns_[^;]+;/g, 'http://ns.adobe.com/Extensibility/1.0/'));
     }
 
+    // Some SVGs exported from Photoshop have been found to have an invalid mime type
+    // Chrome and Safari won't render these SVGs, so we correct it here
+    const invalidMimeType = svgString.match('xlink:href=("|\')data:(img/png)');
```

Comment:
Regexp nit: `["']` is the same as `("|\')` and doesnt use a capture group

Also - I think we should test for just "data:img/png" but be a lot smarter about the replace with a regexp:
```
if (svgString.contains('data:img/png')) {
  svgString = svgString.replace(/(<image[^>]+xlink:href=["'])data:img/png/g, ($0, $1) => `${$1}data:image/png`);
}
```

The detection for "do we need to be specific" should be quick but if we detect it we should target very closely only image xlink:href attributes?